src/datafactory/azext_datafactory/tests/latest/test_datafactory_scenario.py

```diff
@@ -0,0 +1,984 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+import os
+from azure.cli.testsdk import ScenarioTest
+from .. import try_manual, raise_if
+from azure.cli.testsdk import ResourceGroupPreparer
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+@try_manual
+def setup(test, rg):
+    pass
+
+
+# EXAMPLE: /Factories/put/Factories_CreateOrUpdate
+@try_manual
+def step__factories_put_factories_createorupdate(test, rg):
+    test.cmd('az datafactory factory create '
+             '--location "East US" '
+             '--name "{exampleFactoryName}" '
+             '--resource-group "{rg}"',
+             checks=[])

```

Comment:
can you add some value checks?

Reply:
This has not been fixed yet, the outdated status is just because I changed the eof mark.